OpenShift/helm/nexus-iq/values.yaml

```diff
@@ -1,31 +1,23 @@
 # Default values for iqserver.
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
-
+replicaCount: 1
+image:
+  repository: sonatype/nexus-iq-server
+  pullPolicy: IfNotPresent
+  
 iq:
   name: nxiq
-  imageName: registry.connect.redhat.com/sonatype/nexus-iq-server
-  imageTag: 1.85.0-01-ubi
-  imagePullPolicy: IfNotPresent
-  imagePullSecret: ""
+  hostname: iq-server.demo
+  imageName: sonatype/nexus-iq-server
```

Comment:
as you noted, i see here we switched back to the normal image. that seems appropriate for the stand-alone helm chart, but I'm curious to hear @adrianpowell 's opinion on it.  I'd expect we'd just switch it to the certified image when generating the operator from this newer chart.

Reply:
agreed, for a helm chart that ends up in Helm Hub, we want our public image, for operators, we'll want the certified image.